.github/workflows/release-qa-web.yml

```
@@ -3,67 +3,80 @@ name: QA - Web Release
 
 on:
   workflow_dispatch:
-    inputs:
-      image_extension:
-        description: "Image tag extension"
-        required: false
-
-env:
-  _QA_CLUSTER_RESOURCE_GROUP: "bw-env-qa"
-  _QA_CLUSTER_NAME: "bw-aks-qa"
-  _QA_K8S_NAMESPACE: "bw-qa"
-  _QA_K8S_APP_NAME: "bw-web"
 
 jobs:
-  deploy:
-    name: Deploy QA Web
+  setup:
+    name: Setup
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Branch check
+        run: |
+          if [[ "$GITHUB_REF" != "refs/heads/feature/DEVOPS-843_qa_web_vault_to_cloudflare" ]] && [[ "$GITHUB_REF" != "refs/heads/master" ]] && [[ "$GITHUB_REF" != "refs/heads/rc" ]] && [[ $GITHUB_REF != refs/heads/hotfix-rc/* ]]; then
```

Looks like there's some testing code still in here
```suggestion
          if [[ "$GITHUB_REF" != "refs/heads/master" ]] && [[ "$GITHUB_REF" != "refs/heads/rc" ]] && [[ $GITHUB_REF != refs/heads/hotfix-rc/* ]]; then
```